tests/net/stelline/README.md

```diff
@@ -0,0 +1,21 @@
+# Stelline
+
+Stelline is a test runner based around DNS message request/response sequences defined in a human readable text file format.
+
+## Etymology and origins
+
+The Stelline replay file format is a variant of the original `.rpl` format used by the [Unbound](https://www.unbound.net/) test tool "testbound".
```

Comment:
testbound is based on (or inspired by) on ldns-testns

Reply:
Is that better?

Reply:
So it should be 'testbound was originally based on ldns-testns'

Reply:
I made some changes, let me know if it's good now.